src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,12 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (!this.store) {
+                this.store = window.storeService.open(window.name);
+            }
```

Comment:
yes, I spotted that and it was removed in a later commit. However a subsequent commit put it back in as I did end up using it in the end